prometheus/client.go

```diff
@@ -117,6 +125,42 @@ func (in *Client) GetSourceWorkloads(namespace string, namespaceCreationTime tim
 	return routes, nil
 }
 
+func (in *Client) GetDestinationServices(namespace string, namespaceCreationTime time.Time, workloadname string) ([]Service, error) {
+	reporter := "source"
+	if config.Get().IstioNamespace == namespace {
+		reporter = "destination"
+	}
+
+	queryTime := util.Clock.Now()
+	queryInterval := queryTime.Sub(namespaceCreationTime)
+	groupBy := "(destination_service_namespace, destination_service_name, destination_service)"
+	query := fmt.Sprintf("sum(rate(istio_requests_total{reporter=\"%s\",source_workload=\"%s\",source_workload_namespace=\"%s\"}[%vs])) by %s",
```

Comment:
The query used for "GetSourceWorkloads" is
```
delta(istio_requests_total{reporter=\"%s\",destination_service_name=\"%s\",destination_service_namespace=\"%s\"}[%vs])
```
I guess that this one is linked with how the graph is fetching this info, and perhaps in a next step we can change/sync the GetSourceWorkloads (if it makes sense).